example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -130,21 +130,50 @@ struct FillBufferKernel
 auto main()
 -> int
 {
-// This example is hard-coded to use the sequential backend.
-#if defined(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED)
-
+// This example is working if CUDA/HIP is exclusively used
```

Comment:
I am confused. Is the example working or not working for exclusive CUDA / HIP? From the code looks like not.

Reply:
If you use the alpaka exclusive cmake options e.g. `ALPAKA_ACC_GPU_CUDA_ONLY_MODE` or ` ALPAKA_ACC_GPU_CUDA_ONLY_MODE` you will have only a device accelerator but some examples requires a host and a device accelerator.